torch/csrc/jit/codegen/cuda/scheduler/registry.cpp

```diff
@@ -712,59 +725,66 @@ class NormalizationScheduler : public SchedulerEntry {
     scheduleNormalization(fusion, rparams_);
   }
 
-  static bool canSchedule(Fusion* fusion, SchedulerRuntimeInfo& runtime_info) {
+  static bool canSchedule(
```

Comment:
This looks like heavy change but it should really be just re-ordering some lines. Need to go over again to make sure not mistaken.